.github/workflows/ci.yml

```diff
@@ -0,0 +1,25 @@
+name: Sentinel CI
+
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Maybe we could choose an LTS version (e.g. 18.04)? FYI: https://github.com/actions/virtual-environments/issues/1816